app/base/handlers.js

```diff
@@ -67,7 +67,8 @@ exports.logout = (req, res) => {
     }
   });
   req.session.destroy(() => {
+    const encodedReturnTo = encodeURI(`${config.app.protocol}://${config.app.host}:${config.app.port}`);
     res.clearCookie(config.session.name);
-    res.redirect('/');
+    res.redirect(`https://${config.auth0.domain}/v2/logout?returnTo=${encodedReturnTo}&client_id=${config.auth0.clientID}`);
```

Comment:
I would return this value from a self-documenting function (even though it's only used once).
Also, the `/v2/logout` bit looks like it should be in configuration, eg:
 eg:
```javascript
function sso_logout_url() {
  const returnTo = encodeURI(...);
  return `https://${config.auth0.domain}${config.auth0.sso_logout_url}?returnTo=${returnTo}&client_id=${config.auth0.clientID}`;
}
```

Reply:
👍